### PR TITLE
Update gdk_pixbuf.rb

### LINKED ISSUE #1935: Gdk_pixbuf - Post Installation Problem
Hy,

When I Compile It, it installs the 'libraries', but can't finish the 'installation'.
So I removed those lines(from the rb file):

 def self.postinstall
    # If the package is installed using a "DESTDIR" method, an important file was not installed and should be copied or generated.
    system "sudo gdk-pixbuf-query-loaders --update-cache"
  end

And It Installs Successfully, 
 then I do a:      sudo gdk-pixbuf-query-loaders --update-cache

### PR DESCRIPTION
**Pull request by lovedheart**
Fixes #1935

### CODE REVIEW

File: packages/gdk_pixbuf.rb [hunk 1]
```diff
@@ -39,6 +39,6 @@ def self.install
   
   def self.postinstall
     # If the package is installed using a "DESTDIR" method, an important file was not installed and should be copied or generated.
-    system "sudo gdk-pixbuf-query-loaders --update-cache"
+    puts "please run \"sudo gdk-pixbuf-query-loaders --update-cache\" to generate an important file that was not installed."
```
Comment (uberhacker) [inline packages/gdk_pixbuf.rb]: Is the sudo command necessary?  What happens if you exclude it?

Comment (lovedheart) [reply packages/gdk_pixbuf.rb]: It is necessary. If it is excluded, you can also install it without any errors. But it's highly recommended to run this

Comment (uberhacker) [reply packages/gdk_pixbuf.rb]: What exactly does it install and where?  If it only installs in the `/usr/local` directory, the sudo prefix is not needed.

Comment (lovedheart) [reply packages/gdk_pixbuf.rb]: I don't know it exactly. The only problem is that we use DESTDIR. If use regular destdir, there is no such a problem.

Comment (uberhacker) [reply packages/gdk_pixbuf.rb]: I believe this is what you will want to do:
```
   def self.install
     system "pip install six" # for installation process
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system "mkdir -p #{CREW_DEST_LIB_PREFIX}/gdk-pixbuf-2.0/2.10.0"
+    system "touch #{CREW_DEST_LIB_PREFIX}/gdk-pixbuf-2.0/2.10.0/loaders.cache"
     system "pip uninstall --yes six"
   end

   def self.postinstall
-    # If the package is installed using a "DESTDIR" method, an important file was not installed and shoul$
-    puts "please run \"sudo gdk-pixbuf-query-loaders --update-cache\" to generate an important file that $
+    system 'gdk-pixbuf-query-loaders --update-cache'
   end
```

Comment (uberhacker) [reply packages/gdk_pixbuf.rb]: Did you get a chance to test this suggestion yet @lovedheart?

**Comment by lovedheart** [comment]
I've changed the code

**Comment by uberhacker** [comment]
Can you bump the version and remove the binaries please?

**Comment by lovedheart** [comment]
@uberhacker  changed it

**Pull request merged**